File: trunk/C#/Code Templates/C#/fragments/ClassProperty.fragment.cs
		protected TYPE _NAME; // cached lookup value
		public TYPE NAME
		{
			get {
				if(_NAME == null)
					_NAME = dbvos.retrieve(NAMEId, typeof(TYPE)) as TYPE;
				return _NAME;
			}
			set 
			{
				// overrides cached value
				_NAME = value;
			}
		}
		

File: C#/Data Fairy/Data Fairy Editor/Code Templates/PHP/fragments/ClassProperty.fragment.php
	private $_{NAME}; // cached lookup value
	public function get_{NAME}()
	{
		if($this->_{NAME} == null)
			$this->_{NAME} = $this->dbvos->retrieve($this->{NAME}Id, "{TYPE}");
		return $this->_{NAME};
	}
		
	function set_{NAME}($value)
	{
		// overrides cached value
		$this->_{NAME} = $value;
	}
		